workitem/workitemtype_repository.go

```diff
@@ -154,6 +156,18 @@ func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UU
 		return nil, errors.NewInternalError(err.Error())
 	}
 
+	// create relationship between workitemtype and category
+	if categoryID != uuid.Nil {
```

Comment:
@kwk agreed. since other ids use `nil`, its better to do it in the same way for categories. I fixed it here -> 3c967ab